libPhoneNumber/NBPhoneNumberUtil.m

```diff
@@ -3496,15 +3496,26 @@ - (CTTelephonyNetworkInfo *)telephonyNetworkInfo {
 }
 
 - (NSString *)countryCodeByCarrier {
-  NSString *isoCode = [[self.telephonyNetworkInfo subscriberCellularProvider] isoCountryCode];
-
-  // The 2nd part of the if is working around an iOS 7 bug
-  // If the SIM card is missing, iOS 7 returns an empty string instead of nil
-  if (isoCode.length == 0) {
-    isoCode = NB_UNKNOWN_REGION;
-  }
-
-  return isoCode;
+#if __IPHONE_OS_VERSION_MIN_REQUIRED >= 120000
```

Comment:
I think it's cleaner to use 
if (@available(iOS 12, *)) {
  ...
}

Reply:
The bonus to the #if __IPHONE_OS_VERSION_MIN_REQUIRED` check is that it eliminates the build warning anyone gets if they are compiling or targeting a version great than 12.0.

Whereas the `if (@available)` is a runtime if check (as far as I know, at least)